account_asset_management_xls/models/account_asset.py

```diff
@@ -3,9 +3,16 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import api, models
-from odoo.addons.report_xlsx_helper.report.abstract_report_xlsx \
-    import AbstractReportXlsx
-_render = AbstractReportXlsx._render
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from odoo.addons.report_xlsx_helper.report.abstract_report_xlsx \
+        import AbstractReportXlsx
+    _render = AbstractReportXlsx._render
+except (ImportError, IOError) as err:
+    _logger.debug(err)
```

Comment:
Don't you need to define _render in the except block (maybe _render = None) and check when _render is used  that it is not None and then raise a UserError with a friendly error message?

Reply:
`_render` is not use in this file and I don't take time to know cause I don't really how to search :disappointed_relieved:

Reply:
@luc-demeyer do you know if this code can be removed? It looks unused, just want to check if there is no magic involved.

Reply:
_render can be removed here since not called. It could be useful though to keep the line but simply commented out.
As soon as you want to customize the excel report via a custom module (via inherit) than you need the _render function, hence it's good to keep it somehow visible for 'educational' purposes.